charts/abtest-model/values.yaml

```diff
@@ -0,0 +1,18 @@
+namespace: abtesting
+appname: model
+
+deployment:
+  name: model-green  
+  bluegreen: green
+  container:
+    name: model  
+    port: 5001
+  image:
+    name: amlabtest74c95585.azurecr.io/abtestimg
+    tag: 2 
```

Comment:
It seems tag isn't used in your chart.  It's best practice to keep tag vs use just one string for the whole repo:tag.

Reply:
Removed